srcpkgs/sublime-merge/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'sublime-merge'
 pkgname=sublime-merge
-version=2020
+version=2027
 revision=1
 archs="x86_64"
 wrksrc=sublime_merge
 depends="libpng gtk+ hicolor-icon-theme desktop-file-utils"
 short_desc="New Git client, from the makers of Sublime Text"
 maintainer="Adelmo Junior <noblehelm@gmail.com>"
-license="Proprietary"
+license="custom:Proprietary" #no separate file to use with vlicense
```

Comment:
Please try to make something [reproducible][1] out of https://www.sublimehq.com/eula.

[1]: https://github.com/void-linux/void-packages/blob/2de8f963da492cb2bfae6bd74aa0d42bc2f8897b/srcpkgs/spotify/template#L24

Reply:
Nice trick! I wonder if it makes sense for the anydesk and wps-office templates.

Reply:
@Chocimier : Done!

Reply:
> I wonder if it makes sense for the anydesk and wps-office templates.

On update, sure.

Changed to use xbps-fetch instead of curl.